src/ethereum_test/types.py

```diff
@@ -318,14 +394,137 @@ def to_geth_dict(self) -> Mapping[str, Any]:
         return header
 
 
+@dataclass(kw_only=True)
+class Header:
+    """
+    Header type used to describe block header properties in test specs.
+    """
+
+    coinbase: Optional[str] = None
+    parent_hash: Optional[str] = None
+    ommers_hash: Optional[str] = None
+    state_root: Optional[str] = None
+    transactions_root: Optional[str] = None
+    receipt_root: Optional[str] = None
+    bloom: Optional[str] = None
+    difficulty: Optional[int] = None
+    number: Optional[int] = None
+    gas_limit: Optional[int] = None
+    gas_used: Optional[int] = None
+    timestamp: Optional[int] = None
+    extra_data: Optional[str] = None
+    mix_digest: Optional[str] = None
+    nonce: Optional[str] = None
+    base_fee: Optional[int] = None
+    hash: Optional[str] = None
+
+
+@dataclass(kw_only=True)
+class Block(Header):
+    """
+    Block type used to describe block properties in test specs
+    """
+
+    rlp: Optional[str] = None
+    """
+    If set, blockchain test will skip generating the block using
+    `evm_block_builder`, and will pass this value directly to the Fixture.
+
+    Only meant to be used to simulate blocks with bad formats, and therefore
+    requires the block to produce an exception.
+    """
+    rlp_modifier: Optional[Header] = None
+    """
+    An RLP modifying header which values would be used to override the ones
+    returned by the  `evm_transition_tool`.
+    """
+    exception: Optional[str] = None
+    """
+    If set, the block is expected to be rejected by the client.
+    """
+    txs: Optional[List[Transaction]] = None
+    """
+    List of transactions included in the block.
+    """
+    uncles: Optional[List[Header]] = None
```

Comment:
Pedantic, but would be great to switch to `ommers` if it doesn't complicate things too much.

Reply:
Done, at least only in the fields of the types defined in this package.
The json fields for geth could not be changed for example.